src/cohort.js

```diff
@@ -0,0 +1,104 @@
+class Cohort {
+  constructor() {
+    this.cohortList = []
+  }
+
+  createCohort(cohortName) {
+    const newCohort = { name: cohortName, students: [], capacity: 24 }
+    const findCohort = this.cohortList.find(
```

Comment:
you should use searchCohort here to check for the existence as you already defined it and it'd refactor the code.

Reply:
I agree with @AllyDouillette here. Also, try to name variables with nouns and functions with verbs - `findCohort` suggests that this is a function that finds a cohort, `foundCohort` would be better here